### ISSUE TITLE
VOTING: Texts needed to inform admins and participants that admins will see how participants voted

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1649]
**What is the problem?**
@ssin1901 had a user support query from an admin of a GA, wanting to know whether she would be able to see how people voted. I remember this was previously an FAQ that Anett would get from partners. Sara and I also discussed that we don't currently communicate to participants that the admin(s) will see their vote, which seems a very important thing for them to know before voting. 

**Why is this a problem?**
It causes confusion and raises questions from users, and also is a privacy issue (in terms of participants not knowing that their vote will be visible to admins).

**Possible solution.**
@kevincrepin, Sara and I have agreed on the following fixes, but please say if you have any alternative solutions :) 

1. For admins, we can add a text in the vote settings here: saying “For both options, only the topic admin(s) can see how people voted, once the vote has ended.”

![image](https://github.com/user-attachments/assets/f0fc1f01-ad0b-4bbf-b90a-b92be3d3e047)

2. For admins and participants, we can add a line here: “Only the topic admin(s) can see how people voted.”

![image](https://github.com/user-attachments/assets/8761ba04-c3f9-4037-b418-53f84a2ed3c1)

3. For participants, we can add a line here, saying: “Only the topic admin(s) will see how you voted.”

![image](https://github.com/user-attachments/assets/86e0e8e5-5b0b-41ef-a9e8-465e674ae02a)

**Comment by BeccaMelhuish** [comment]
I'll leave this for a while for input from @kevincrepin, but I think I can update these in the text strings myself when it comes to it :)

Agreed with @ssin1901 that this should go to 'next?'

**Comment by kevincrepin** [comment]
I see no problems with this :)

**Comment by BeccaMelhuish** [comment]
OK great, have edited in the translation strings, moving to 'in progress' for @ilmartyrk to merge :)

**Comment by ssin1901** [comment]
@BeccaMelhuish seems to be an okay fix (screenshot1). If you agree, then let's move to Pre Live?

<img width="853" alt="Screenshot 2024-12-10 at 12 08 27" src="https://github.com/user-attachments/assets/4928cd21-596c-49df-b8c1-567f0f75005f">

**Comment by BeccaMelhuish** [comment]
@ssin1901 yes looks good and will move to pre-live, though @ilmartyrk ideally this second sentence would be in regular not in bold :)  

![image](https://github.com/user-attachments/assets/78b1386c-980d-423d-bd3f-67a1d1986704)

**Comment by BeccaMelhuish** [comment]
It's still bold but moving to pre-live column as it's not that important :)
